src/components/bookmark/bookmark.js

```diff
@@ -0,0 +1,75 @@
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { Card } from 'react-bootstrap';
+import { bindActionCreators } from 'redux';
+import * as fechMybookmark from '../../redux/actions/bookmark/fetchBookmark';
+
+export const Bookmark = (props) => {
+  const [myBookmarks, setBookmarks] = useState([]);
+  const [loading, setLoading] = useState(true);
+  const { bookmarks, actions, isSuccess } = props;
+  useEffect(() => {
+    actions.fechMybookmark();
+  }, []);
+  useEffect(() => {
+    if (isSuccess === 'success') {
+      setBookmarks(bookmarks);
+      setLoading(false);
+    }
+    if (isSuccess === 'fail') {
+      window.location.replace('/');
+    }
+  }, [props.setBookmarks, props.isSuccess]);
+
+  if (loading) {
+    return (
+      <p>
+        Loading...
+      </p>
+    );
+  }
+  return myBookmarks.map((bookmark) => {
```

Comment:
Is it possible for you to use listOfArticles component to display the bookmarks

Reply:
yes, it is but I don't think that big article cards would be good for bookmarks list. let me know your concern